container/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Use the official lightweight Python image.
+# https://hub.docker.com/_/python
+FROM python:slim
+
+# Copy local code to the container image.
+ENV APP_HOME /app
+WORKDIR $APP_HOME
+
+# Install production dependencies.
+RUN pip install fediblockhole
+
+USER 1001
+# Run the web service on container startup. Here we use the gunicorn
+# webserver, with one worker process and 8 threads.
+# For environments with multiple CPU cores, increase the number of workers
+# to be equal to the cores available.
```

Comment:
This comment block doesn't seem to refer to what's happening here. Leftover from a cut+paste from elsewhere?

Reply:
Uh, probably...